.github/workflows/build.yml

```diff
@@ -309,6 +309,13 @@ jobs:
           ./admin/pull-readme
           git diff --exit-code
 
+      - name: Spell check code
+        uses: crate-ci/typos@85f62a8a84f939ae994ab3763f01a0296d61a7ee
+        with:
+          config: .github/typos.toml
+          files: examples/ provider-example/ rustls/ rustls-post-quantum/
```

Comment:
Some of the earlier commits are fixing typos in places not covered here, like `bogo`. Is it worth including that dir to avoid backsliding?

Reply:
eliminated this list, and added some excludes to stop it looking at test vectors, machine-generated svgs, etc.